README.md

```diff
@@ -171,6 +171,36 @@ You can view the log output using `docker logs -f` or by passing `-it` to `docke
 
 Congratulations, you now have a working iRODS HTTP API server!
 
+#### Validating the Configuration
+
+If you have Python 3 and the jsonschema module installed, you can validate your configuration on startup. To do that, launch the container with `--jsonschema-file`. Below is an example.
```

Comment:
Reword/review this section. The first statement is confusing.

This is the server's python environment.

Reply:
Done. Let me know if it needs to be tweaked more.